Cargo.toml

```
@@ -19,6 +19,9 @@
 resolver = "2"
 members = ["crates/catalog/*", "crates/iceberg", "crates/test_utils"]
 
+[workspace.package]
+homepage = {url = "https://rust.iceberg.apache.org/"}
```

I noticed that the CI is failing:
```
Caused by:
  TOML parse error at line 23, column 12
     |
  23 | homepage = {url = "https://rust.iceberg.apache.org/"}
     |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  invalid type: map, expected a string
```

```suggestion
homepage = "https://rust.iceberg.apache.org/"
```

---

I have fixed this in the new PR , please review those changes and let me know if something has to be improved